lib/rubocop/cop/rspec/factory_bot/consistent_parentheses_style.rb

```diff
@@ -0,0 +1,99 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      module FactoryBot
+        # Use a consistent style for parentheses in factory bot calls
+        #
+        # @example
```

Comment:
As a last touch, I suggest splitting this into two examples, one per style, like e.g. [here](https://github.com/rubocop-hq/rubocop-rspec/blob/master/lib/rubocop/cop/rspec/message_spies.rb#L11).